src/Provider/PointCloudProvider.js

```diff
@@ -272,3 +274,7 @@ export default {
         });
     },
 };
+
+export const _testing = {
+    parseMetadata,
```

Comment:
I'm not okay with this, please move it to the default export, even if I understand that this is for testing purpose only.

Reply:
Thanks for your review. Can you elaborate on the reasons you don't like this?

Reply:
To elaborate on my position, I'm pretty against exposing this to the default export, because it's a private method that has no reason to be exposed. Once you expose such a method, it becomes part of the public api, which means we have an implicit commitment to it. I know that this way of thinking is not very common on this project, but I'd like not to introduce api breakage when we can avoid it. For me `_testing` is a good middle-ground as it conveys the fact it is a private method.

Reply:
I don't really know, maybe in fact it is a good idea. We could agree on a special exported namespace we could use to test internal methods. So in fact maybe keep it and find a more suitable name ? `_testing` feels muddled to me.

Reply:
> Once you expose such a method, it becomes part of the public api

Providers are not in the public API so no problems here, but see my change of pov above ;)

Reply:
> Providers are not in the public API so no problems here

Just a quick note on this: I'm afraid they *are*. Everything importable from a ES6 configuration is part of the public API and we should treat it that way. Every transpiled file in `lib` is importable. Actually, in my opinion this should be the preferred way of using iTowns. The 2 `Main` files are just convenience.

> So in fact maybe keep it and find a more suitable name ? _testing feels muddled to me.

I don't know, `testing` seemed a suitable name for an export for testing purpose... I don't see a better name (yet).

Reply:
Ok I see one possible solution: surround testing exports with `__DEBUG__` tests. But I need to add support for this in our `.babelrc`. So as not to enter into a rabbit hole for this PR, I suggest to keep it that way here and revisit it later. Is it acceptable for you @zarov?

Reply:
(and then rename `__DEBUG__` with `__DEV__`)

Reply:
(sorry for being so undecided) yeah `testing` is good :D But if you feel that surrounding testing with those is better, let's go for it.